cmd/util/cols/history.go

```diff
@@ -34,7 +35,7 @@ var (
 		ColumnConfig: table.ColumnConfig{Name: "Exec. ID", WidthMax: 10, WidthMaxEnforcer: text.WrapText},
 		Value: func(j *models.JobHistory) string {
 			if j.ExecutionID == "" {
-				return "/"
+				return strings.Repeat("-", 10)
```

Comment:
might want to extract `10` to a constant given it must match the column max width